src/interceptors/XMLHttpRequest/XMLHttpRequestOverride.ts

```diff
@@ -38,6 +38,11 @@ export const createXMLHttpRequestOverride = (
     _events: XMLHttpRequestEvent<XMLHttpRequestEventTargetEventMap>[] = []
 
     /* Request state */
+    public static readonly UNSENT = 0
+    public static readonly OPENED = 1
+    public static readonly HEADERS_RECEIVED = 2
+    public static readonly LOADING = 3
+    public static readonly DONE = 4
     public readonly UNSENT = 0
     public readonly OPENED = 1
     public readonly HEADERS_RECEIVED = 2
```

Comment:
We cannot mark the existing properties as static, because states should be available on both class instance and as a class static property, right?

```js
XMLHttpRequest.DONE
const req = new XMLHttpRequest()
req.DONE
```

Reply:
Yeap, that seems to be the case. Awesome!